full/Dockerfile

```
@@ -108,7 +108,9 @@ RUN curl -s "https://get.sdkman.io" | bash \
              && sdk install java 11.0.2-open \
              && sdk default java 8.0.201-oracle \
              && sdk install gradle \
-             && sdk install maven"
+             && sdk install maven \
+             && mkdir /home/gitpod/.m2 \
+             && echo '<settings><localRepository>/workspace/m2-repository/</localRepository></settings>' > /home/gitpod/.m2/settings.xml"
```

Nit: Maybe add whitespaces to the XML file? Here is a suggestion (I use `printf` because it's safer than `echo` with escaped characters and CLI options across various shells):
```suggestion
             && printf '<settings>\n  <localRepository>/workspace/m2-repository/</localRepository>\n</settings>\n' > /home/gitpod/.m2/settings.xml"
```